subsys/logging/Kconfig

```diff
@@ -330,18 +330,45 @@ config LOG_BACKEND_RTT_MODE_DROP
 config LOG_BACKEND_RTT_MODE_BLOCK
 	bool "Block until message is transferred to host."
 	help
-	  Waits until there is enough space in the up-buffer for a message
-	  and until the message is completely transferred to the host.
+	  Waits until there is enough space in the up-buffer for a message.
 
 endchoice
 
+if LOG_BACKEND_RTT_MODE_DROP 
+
 config LOG_BACKEND_RTT_MESSAGE_SIZE
 	int "Size of internal buffer for storing messages."
 	range 32 256
 	default 128
 	help
 	  This option defines maximum message size transferable to up-buffer.
 
+endif #LOG_BACKEND_RTT_MODE_DROP
+
+if LOG_BACKEND_RTT_MODE_BLOCK
+
+config LOG_BACKEND_RTT_BUFFER_SIZE
+	int "Size of the output buffer"
```

Comment:
This is not the RTT buffer? Please clarify.

Reply:
added help, renamed to `LOG_BACKEND_RTT_OUTPUT_BUFFER_SIZE`